itou/nexus/management/commands/populate_metabase_nexus.py

```diff
@@ -0,0 +1,256 @@
+
+    def populate_memberships(self):
+        employers_qs = (
+            CompanyMembership.objects.active().select_related("company").only("company__uid", "user_id", "is_admin")
+        )
+        prescribers_qs = (
+            PrescriberMembership.objects.active()
+            .select_related("organization")
+            .only("organization__uid", "user_id", "is_admin")
+        )
+
+        def serializer(membership):
+            if isinstance(membership, CompanyMembership):
+                uniq_id = f"{SOURCE}--{membership.company.uid}"
+            else:
+                uniq_id = f"{SOURCE}--{membership.organization.uid}"
+
+            return [
+                SOURCE,
+                f"{SOURCE}--{membership.user_id}",
+                uniq_id,
+                "administrateur" if membership.is_admin else "collaborateur",
+                self.run_at,
+            ]
+
+        columns = {
+            "source": "text",
+            "user_id_unique": "text",
+            "structure_id_unique": "text",
+            "role": "text",
+            "updated_at": "timestamp with time zone",
+        }
+
+        populate_table(MEMBERSHIPS_TABLE, columns, serializer, [employers_qs, prescribers_qs])
+
+    def populate_structures(self):
+        prescribers_qs = PrescriberOrganization.objects.select_related("insee_city")
+        company_qs = Company.objects.select_related("insee_city")
+
+        def serializer(org):
+            if isinstance(org, Company):
+                org_name = org.display_name
+                org_kind = f"company--{org.kind}"
+            else:
+                org_name = org.name
+                org_kind = f"prescriber--{org.kind}"
+
+            return [
+                SOURCE,
+                str(org.pk),
+                f"{SOURCE}--{org.uid}",
+                org.siret,
+                org_name,
+                org_kind,
+                org.insee_city.code_insee if org.insee_city else None,
+                org.address_on_one_line,
+                org.post_code,
+                org.latitude,
+                org.longitude,
+                org.email,
+                org.phone,
+                self.run_at,
+            ]
+
+        columns = {
+            "source": "text",
+            "id_source": "text",
+            "id_unique": "text",
+            "siret": "text",
+            "nom": "text",
+            "type": "text",
+            "code_insee": "text",
+            "adresse": "text",
+            "code_postal": "text",
+            "latitude": "double precision",
+            "longitude": "double precision",
+            "email": "text",
+            "téléphone": "text",
+            "updated_at": "timestamp with time zone",
+        }
+
+        populate_table(STRUCTURES_TABLE, columns, serializer, querysets=[prescribers_qs, company_qs])
+
+    def add_arguments(self, parser):
+        parser.add_argument("--reset-tables", action="store_true", default=False, help="Reset the table schema")
```

Comment:
```suggestion
        parser.add_argument("--reset-tables", action="store_true", help="Reset the table schema")
```

Inutile avec `store_true`